### PR TITLE
Have a command to add all labels used in the process #9

### PR DESCRIPTION
**Pull request by LowWeiLin**
Creates status labels. Not tested.

### CODE REVIEW

**Comment by kanghj** [comment]
@LowWeiLin fix your merge conflicts

**Comment by dariusf** [comment]
Need to change [this](https://github.com/HubTurbot/HubTurbot/blob/master/.hubturbot) to use `s` also

**Comment by LowWeiLin** [comment]
Changes MADE @kanghj

**Comment by kanghj** [comment]
The default status prefix in #16 is <code>status.</code>

**Comment by LowWeiLin** [comment]
oh wait so i should use the config to determine the prefix?

File: server.js [hunk 1]
```diff
@@ -175,7 +175,51 @@ function applyToReview(config, user, repo, number) {
...
+    user: data.repository.owner.login,
+    repo: data.repository.name,
+    name: "s.ToMerge",
+    color: "38923D"
+  });
+}
+
 
   // HubTurbot will not reply to itself
   if (data.comment.user.login === "HubTurbot") {
```
Comment (kanghj) [inline server.js:234]: Should we store "HubTurbot" in the config?

**Comment by dariusf** [comment]
I mean, since we are using `s` as the default for this repo, should change our own config file

**Comment by LowWeiLin** [comment]
But we should use the config file to determine the prefix right?

File: server.js (hunk 1 above)
Comment (LowWeiLin) [inline server.js:234]: I think we can create a separate issue for this.

**Comment by LowWeiLin** [comment]
Can I make the config object a global variable? passing it everywhere is a pain.

**Comment by kanghj** [comment]
Nah, global variables are evil

**Comment by LowWeiLin** [comment]
..i'll hardcode to use status. instead then.

**Comment by LowWeiLin** [comment]
Updated @kanghj @dariusf

**Comment by LowWeiLin** [comment]
you alive?

**Comment by LowWeiLin** [comment]
@hubTurbot create all labels please

**Comment by LowWeiLin** [comment]
@HubTurbot create all labels please

**Comment by LowWeiLin** [comment]
@HubTurbot create all labels please

**Pull request merged**